lib/src/resultset-table/ResultsetTable.test.js

```diff
@@ -218,6 +219,15 @@ const rows2 = [
   ],
 ];
 
+const checkboxRows = [
+  [ { displayValue: <DxcCheckbox checked /> }, { displayValue: "Peter" }, { displayValue: "Miami" }],
+  [ { displayValue: <DxcCheckbox checked /> }, { displayValue: "Louis" }, { displayValue: "London" }],
+  [ { displayValue: <DxcCheckbox checked /> }, { displayValue: "Lana" }, { displayValue: "Amsterdam" }],
+  [ { displayValue: <DxcCheckbox /> }, { displayValue: "Rick" }, { displayValue: "London" }],
+  [ { displayValue: <DxcCheckbox /> }, { displayValue: "Mark" }, { displayValue: "Miami" }],
+  [ { displayValue: <DxcCheckbox /> }, { displayValue: "Cris" }, { displayValue: "Paris" }],
+];
+
```

Comment:
i am not sure this visual test is necessary since it does not test the scenario that this pr is trying to fix.

Reply:
You are right, the visual part should simply be checked in Cromatic. I could add a test that checks the value of the key in the <tr> using JSDOM but it seems a bit overkill and unneeded.